web3rpc/rpc-specs/code-samples/javascript/admin/nodeConfig.js

```diff
@@ -0,0 +1,10 @@
+const OpenSdk = require("opensdk-javascript");
```

Comment:
You can refactor the javascript code-samples based on https://github.com/klaytn/web3klaytn/blob/dev/ethers-ext/example/rpc/rpc.js?